frontend/class-monthly-report.php

```diff
@@ -220,7 +220,7 @@
                     var year = $('#year option:selected').text();
                     var grade = $('#grade option:selected').text();
                     var selectedClass = $('#classDropdown option:selected').text();
-                    month = "0" + (parseInt(month) + 1);
```

Comment:
So this would not add the '0' to months before October. 
1-9 months should be like 01, 02...

Reply:
I was thinking to put an if else condition in between to resolve the conflict like:
 if parseInt(month)<10:
                    month = "0"+(parseInt(month) + 1);
                   else: 
                    month = (parseInt(month) + 1);


If this helps, I'll make the final changes, lemme know!

Reply:
This would be better right? 
Add `var` as well.
`var month = parseInt(month) < 10 ? "0" + (parseInt(month) + 1) : (parseInt(month) + 1)`

Reply:
cool, making changes right away:)